dub.json

```diff
@@ -13,7 +13,7 @@
     "stringImportPaths": ["resources"],
 
     "preGenerateCommands": [
-        "git describe --tags > resources/VERSION"
+        "cd $PACKAGE_DIR && git describe --tags > resources/VERSION"
```

Comment:
* If I recall correctly, on Windows the `%VAR%` syntax is used to interpolate environment variables
* I don't think this would work if `$PACKAGE_DIR` contains spaces